ScheduleSetting/ScheduleTable/story.jsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import {
+  action,
+  storiesOf,
+} from '@kadira/storybook';
+import { checkA11y } from 'storybook-addon-a11y';
+import ScheduleTable from './index';
+
+const times = [
+  {
+    value: {
+      hours: 9,
+      minutes: 41,
+    },
+    onChange: action('on-change'),
+  },
+  {
+    value: {
+      hours: 19,
+      minutes: 0,
+    },
+    onChange: action('on-change'),
+  },
+];
+
+const timesSingle = [
+  {
+    value: {
+      hours: 9,
+      minutes: 41,
+    },
+    onChange: action('on-change'),
+  },
+];
+
+const timesEmpty = [];
+
+const days = [
```

Comment:
Looks like there's quite a bit of duplicate code here. This will be easier to maintain if you use a loop to generate the test data :smile: 

```js
const daysOfWeek = [ 'Monday', 'Tuesday'/* etc */];
const days = [];
daysOfWeek.forEach((dayName) => {
  days.push({
   dayName,
   postingTimesTotal: times.length,
   times,
  });
  // generate other test day here
});
```